src/main/kotlin/com/ampnet/identityservice/persistence/repository/MailTokenRepository.kt

```diff
@@ -7,4 +7,5 @@ import java.util.UUID
 interface MailTokenRepository : JpaRepository<MailToken, Int> {
     fun findByToken(token: UUID): MailToken?
     fun findByUserAddressOrderByCreatedAtDesc(address: String): List<MailToken>
+    fun findByUserAddress(address: String): MailToken?
```

Comment:
This query is not valid because user can have multiple tokens for the same address. Use `findByUserAddressOrderByCreatedAtDesc`.